src/amazonica/core.clj

```diff
@@ -284,13 +298,11 @@
          (fmap str/capitalize)
          str/join)))
 
-
 (defn- aws-package?
   [clazz]
   (->> (.getName clazz)
        (re-find #"com\.amazonaws\.services")
-       nil?
-       not))
+       some?))
```

Comment:
This is not germane to the PR, but I saw it, so I fixed it. (Per its own docstring, `some?` is considered more idiomatic.)